package.json

```diff
@@ -25,7 +25,7 @@
     "compile": "tsc -p .",
     "compile:module": "tsc -p . --outDir ./dist/es --target es2015 --module es2015",
     "compile:rollup": "rollup -c rollup.config.js",
-    "compile:webpack": "webpack --mode production",
+    "compile:webpack": "NODE_OPTIONS=--openssl-legacy-provider webpack --mode production",
```

Comment:
I added details in the ticket, but the short version is that webpack v4 uses MD4 for hashing and versions of Node newer than 16 include OpenSSL, which doesn't allow that by default.